src/components/Header/HeaderNav/HeaderNav.js

```diff
@@ -1,10 +1,85 @@
 import React from 'react';
+import { FormattedMessage } from 'react-intl';
 
-const HeaderNav = (props) => {
+
+const HeaderNav = () => {
   return (
     <nav>
       <ul>
-        <li>{props.user.Email}</li>
+        <li>
```

Comment:
Maybe this could be an iteration, what do you think? To avoid repeat html code. Maybe this information is neccesary to be obtained by the BE so in this case I was thinking that may be we could use a map or somethink like that.

Reply:
Se puede hacer una iteración con react, incluso hardcodeando los datos de entrada, igual no se si vale la pena.

Reply:
i created this pr to discuss that. i talked with @cbernat and maybe is better this is static and not dinamyc. but can changed. What do you think @andresmoschini ?

Reply:
I didn't get exactly what you were talking about, but in first instance maybe is better to consume data as it is in doppler. And then we will see, how can we improve from there.

Reply:
> Maybe this information is neccesary to be obtained by the BE

I did not know that you had so rich object in Doppler, _mainMenuData_, I understand now, and it makes sense absolutely.

Thanks @lucianaDilizio!